internal/cli/atlas/deployments/list.go

```diff
@@ -185,23 +183,27 @@ func ListBuilder() *cobra.Command {
 		},
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			opts.config = config.Default()
-			opts.credStore = config.Default()
+			opts.CredStore = config.Default()
+			log.SetWriter(cmd.OutOrStdout())
 
 			if err := opts.PreRunE(
 				opts.initStore(cmd.Context()),
 				func() error { return opts.defaultSetter.InitStore(cmd.Context()) },
-				opts.InitOutput(cmd.OutOrStdout(), listTemplate)); err != nil {
+				opts.InitOutput(log.Writer(), listTemplate)); err != nil {
 				return err
 			}
 
 			opts.defaultSetter.OutWriter = cmd.OutOrStdout()
 
-			opts.podmanClient = podman.NewClient(log.IsDebugLevel(), log.Writer())
-			return opts.podmanClient.Ready(cmd.Context())
+			opts.PodmanClient = podman.NewClient(log.IsDebugLevel(), log.Writer())
+			return nil
```

Comment:
why did we remove `opts.podmanClient.Ready(cmd.Context())`

Reply:
for all deployments commands, we won't want to fail if Podman is not ready considering that we'll also try to search for  Atlas deployments

Reply:
Thanks for the clarification